RequestManagementSystem/Client/Request.py

```diff
@@ -370,7 +370,7 @@ def Status( self, value ):
     # If the status moved to Failed or Done, update the lastUpdate time
     if value in ( 'Done', 'Failed' ):
       if value != self._Status:
-        self.LastUpdate = datetime.datetime.utcnow().replace( microsecond = 0 )
```

Comment:
Oh yes right !! Well, anyway, I will get ride of this hybrid_property very very soon. I don't want to expose SQLAlchemy on the client side if possible.